libcontainer/configs/mount.go

```diff
@@ -22,6 +28,9 @@ type Mount struct {
 	// Relabel source if set, "z" indicates shared, "Z" indicates unshared.
 	Relabel string `json:"relabel"`
 
+	// Extensions are additional flags that are specific to runc.
+	Extensions int `json:"extensions"`
```

Comment:
This is internal. The option is passed as `copyup` in mount options in the config json.